NuRadioReco/modules/RNO_G/stationLDAVariables.py

```diff
@@ -184,33 +185,4 @@ def coherent_sum(self, event, station, ref_ch):
             
             aligned_wf = np.roll(ch.get_trace(), lag)
             self.sum_chan += aligned_wf
```

Comment:
I do not like the fact that `sum_chan` is implemented as a class member variable. Those variables are typically used to share data across events. This should not happen here right?